cmd/eksctl-anywhere/cmd/upgradeplancluster.go

```diff
@@ -89,13 +91,36 @@ func (uc *upgradeClusterOptions) upgradePlanCluster(ctx context.Context) error {
 		return err
 	}
 
-	componentChangeDiffs := eksaupgrader.EksaChangeDiff(currentSpec, newClusterSpec)
-	if componentChangeDiffs == nil {
-		componentChangeDiffs = &types.ChangeDiff{}
+	componentChangeDiffs := &types.ChangeDiff{}
+
+	if newClusterSpec.Cluster.IsSelfManaged() {
+		client, err := deps.UnAuthKubeClient.BuildClientFromKubeconfig(managementCluster.KubeconfigFile)
+		if err != nil {
+			return err
+		}
+
+		currentManagementComponentsVersionsBundles, err := cluster.VersionsBundleForManagementComponents(ctx, client, currentSpec.Cluster)
+		if err != nil {
+			return err
+		}
+
+		newVersionsBundle := newClusterSpec.FirstVersionsBundle()
+
+		for _, vb := range newClusterSpec.Bundles.Spec.VersionsBundles {
+			fmt.Printf("%s ", vb.KubeVersion)
+		}
+		fmt.Print("\n")
+
+		fmt.Printf("Current vSphere: %s\n", currentManagementComponentsVersionsBundles.VSphere.Version)
```

Comment:
Using logger here may be more consistent with other logs.

Reply:
This was for testing purposes, so I'll clean this up.